src/System.DirectoryServices/tests/System/DirectoryServices/ActiveDirectory/DomainControllerTests.cs

```diff
@@ -39,10 +39,10 @@ public void GetDomainController_InvalidName_ThrowsActiveDirectoryObjectNotFoundE
 
         [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Access to path is denied when in App container")]
-        public void GetDomainController_InvalidIPV6_ThrowsInvalidCastException()
+        public void GetDomainController_InvalidIPV6_ThrowsActiveDirectoryObjectNotFoundException()
         {
             var context = new DirectoryContext(DirectoryContextType.DirectoryServer, "[::1]:port");
-            Assert.Throws<InvalidCastException>(() => DomainController.GetDomainController(context));
+            Assert.Throws<ActiveDirectoryObjectNotFoundException>(() => DomainController.GetDomainController(context));
```

Comment:
Marked anything taking longer than 4 seconds in the automated run as `[OuterLoop]` and as `[OuterLoop("Takes too long on domain joined machines")]` for the ones from my domain joined box, not that these tests are all passing on a domain joined box but I will track those on a separate item.